### ISSUE TITLE
ipynb not working

### ISSUE THREAD

**Issue by lalligagger** [microsoft/vscode-jupyter#6369]
Issue Type: <b>Bug</b>

I have run into a few times now that ipynb notebooks will be working, and then with no apparent change to VS Code I become unable to open.
The loading bar under the filename keeps looping on repeat, and I do not see any of the notebook buttons load.
Previously I've been able to fix this by uninstalling/ reinstalling the Python and Jupyter extensions, but now I am seeing this problem persistently.

VS Code version: Code 1.57.1 (Universal) (507ce72a4466fbb27b715c3722558bb15afa9f48, 2021-06-17T13:28:32.912Z)
OS version: Darwin x64 19.6.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz (8 x 2000)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|10, 7, 3|
|Memory (System)|16.00GB (0.21GB free)|
|Process Argv|-psn_0_45067 --crash-reporter-id f8c463f6-55d2-4646-a41a-4308c19c2549|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-docker|ms-|1.14.0
python|ms-|2021.6.944021595
vscode-pylance|ms-|2021.6.2
jupyter|ms-|2021.6.999406279
remote-containers|ms-|0.183.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551cf:30311713
vspre833:30321513
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255:30323308
vscorehov:30309549
vscod805:30301674
binariesv615:30325510
vsccppwtct:30322537

```

</details>

**Comment by rchiodo** [comment]
Thanks for the bug. 

Can you copy the contents of the 'Help | Toggle Developer Tools' and the 'Jupyter' output tab to this bug? There's likely some indication of the failure in those two console logs.

**Comment by mccrinbc-ss** [comment]
Hey! I'm having the same issue with Anaconda 1.10, vscode 1.57.1, MacOS BigSur 11.3.1. Are these the outputs that you're looking for? 

Jupyter:
User belongs to experiment group 'pythonJediLSP'
User belongs to experiment group 'jupyterTest'
User belongs to experiment group 'pythonDiscoveryModuleWithoutWatcher'
User belongs to experiment group 'pythonSurveyNotificationcf'
User belongs to experiment group 'jupyterEnhancedDataViewer'
User belongs to experiment group 'pythonDeprecatePythonPath'
User belongs to experiment group 'NativeNotebookEditor'
Info 2021-06-29 15:49:18: ZMQ install verified.
Info 2021-06-29 15:49:18: Attempting to start a server because of preload conditions ...

Developer Tools: 
![image](https://user-images.githubusercontent.com/83707037/123860139-841bd080-d8f3-11eb-9f6d-5e09619ed93b.png)

As a sidenote: vscode was working perfectly until I tried changing _Python > Formatting : Provider_ to **black** instead of the default **autopep8**. Even reverting back to default settings, this issue persists. 

Any information to solve this would be great!

**Comment by rchiodo** [comment]
@mccrinbc-ss can you describe your problem more? Is it the same where ipynb files don't open and get stuck on infinite load?

I would have expected a lot more jupyter output. 

Can you share a screenshot?

**Comment by mccrinbc-ss** [comment]
@rchiodo Before attempting anything, the figure below shows what my vscode environment looks like. 

![image](https://user-images.githubusercontent.com/83707037/123864736-183c6680-d8f9-11eb-94f3-dc5db8ccc304.png)

After playing around, I found that by navigating to the file as shown in the figure below, I was able to re-open the .ipynb file without issues. 
![image](https://user-images.githubusercontent.com/83707037/123864959-56398a80-d8f9-11eb-97a4-41dab3137a53.png)

I do need to mention, however, when I wrote the shown scripts, the formatting of vscode was slightly different. It appears as if vscode updated in background and changed the UI only _after_ I attempted to change my Python Formatting. (To be more specific, the Code+ and Markdown+ boxes were not there before, and the cell formatting seems different). 

![image](https://user-images.githubusercontent.com/83707037/123865389-dcee6780-d8f9-11eb-911d-0e5b18dab92b.png)

Is a background update possible? Could I have been on an older version of vscode without knowing it and my scripts were unable to open?

**Comment by rchiodo** [comment]
I think background updating is a setting, but not positive. And yes we're rolling out a new UI to a subset of customers. What you're seeing is the new UI for notebooks.

Are you saying all of your notebooks reopen fine if you reload them? We might need to force this on update.

**Comment by mccrinbc-ss** [comment]
After closing the tabs, reloading them seems to bring me to this new UI without issue. It might be helpful to have this occur on update to avoid confusion.

**Comment by greazer** [comment]
This problem has been resolved in Stable v1.59 and Insiders.
